mlxtend/regressor/linear_regression.py

```diff
@@ -69,6 +74,7 @@ def __init__(self, eta=0.01, epochs=50,
         self.random_seed = random_seed
         self.print_progress = print_progress
         self._is_fitted = False
+        self.method = method
 
```

Comment:
I suggest adding a check for makes things more clear to the user.
I.e., something along the lines of

```python
if method != 'sgd' and minibatches is not None:
    raise ValueError('Minibatches should be set to `None` '
                     'if `method != 'sgd'. Got method='%s'.` % (method))